sdlib/d/gc/util.d

```diff
@@ -288,3 +288,56 @@ unittest alignUp {
 		}
 	}
 }
+
+// FIXME: detect that we're actually on a little-endian machine:
+
+ushort loadBigEndian(void* ptr) {
```

Comment:
It seems sensible to make these store/load templates. They are generally useful and belong in util.